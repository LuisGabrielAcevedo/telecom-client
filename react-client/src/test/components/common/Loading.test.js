import React from "react";
import "@testing-library/jest-dom";
import { shallow } from "enzyme";
import Loading from "src/components/common/Loading";

describe("Loading Component test", () => {
  const wrapper = shallow(<Loading repeat={4} />);
  it("the component should be initialized", () => {
    expect(wrapper).toMatchSnapshot();
  });

  it("the component should render 4 loading", () => {
    expect(wrapper.find(".loading")).toHaveLength(4);
  });
});
