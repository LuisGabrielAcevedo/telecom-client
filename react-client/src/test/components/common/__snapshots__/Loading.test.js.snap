// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading Component test the component should be initialized 1`] = `
Array [
  <div
    className="loading"
    key="0"
  />,
  <div
    className="loading"
    key="1"
  />,
  <div
    className="loading"
    key="2"
  />,
  <div
    className="loading"
    key="3"
  />,
]
`;
